### ISSUE TITLE
Ubuntu crio 1.15, the network config is not loaded automatically into /etc/cni/net.d

### ISSUE THREAD

**Issue by abhijitherekar** [cri-o/cri-o#2739]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**ON Ubuntu machines with kubeadm and crio the cluster nodes never comeup**

**Steps to reproduce the issue:**
1. `systemctl start crio`
2. `kubeadm init --pod-network-cidr=10.244.0.0/16 --apiserver-advertise-address=$IP --cri-socket=/var/run/crio/crio.sock`
3. `kubectl apply network_config.yaml`

**Describe the results you received:

I saw the nodes never come up

```
kubectl get nodes
NAME                      STATUS     ROLES    AGE   VERSION
abhi-sandbox-crio         NotReady   master   32m   v1.15.3
abhi-sandbox-crio-slave   NotReady   <none>   22m   v1.15.2
```
**

**Describe the results you expected

I expected the nodes to be in ready state:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `crio --version`:
`crio --version
crio version 1.15.2-dev`
**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by abhijitherekar** [comment]
fyi @haircommander

**Comment by haircommander** [comment]
As mentioned here: https://github.com/cri-o/cri-o/blob/master/tutorials/kubeadm.md#configuring-cni, `kubeadm` clears CRI-O's default network dir (it seems on startup now, not only on a `kubeadm reset` call).

The workaround is described in that tutorial, where users have to specify a different network dir. Perhaps the directions aren't clear enough? 

Any other fix would be to change the default, or change `kubeadm`'s behaviour.

**Comment by Nefelim4ag** [comment]
I try use cricli runp on static pod with hostNetwork: true
That not work with same error.
CNI not required for hostNetwork.

**Comment by jdanekrh** [comment]
The tutorial looks fine now on second (or maybe third) reading.

When I saw it the first time though, I completely failed to realize that I want to make change to cri-o config. I read the "it is good to change" paragraph and just went on, without acting on it. I also did not set the kubelet options. I somehow completely skipped that section. Afaik I did not have any problems due to that, at least nothing I could see. Maybe if I went further in the setup, I'd realize something is broken...

Only thing I did was kubeadm init --pod-network-cidr, or actually I added the equivalent into kubeadm config file.
